content/apps/quotas.md

```diff
@@ -0,0 +1,123 @@
+---
+menu:
+  main:
+    parent: apps
+title: Quotas
+weight: -10
+---
+
+#### Quotas:
+
+Cloud Foundry capacity is billed by quota. Quotas provide a not to exceed reservation of memory, compute, application routes and service instances. Quotas are associated with and billed to a project IAA number or equivalent billing tag. Quota capacity is a hard limit, unused capacity does not ‘roll over’ month to month.
+
+##### Quota Billing:
+
+Quotas are billed by the unit and tallied daily. A monthly bill might include 9 days at 1 unit, 11 days at 2 units and 10 days at 4 units.
+
+##### Quota Units:
+
+Each unit provides.
+
+- 10 application routes.
```

Comment:
As I see it, the number of routes should be < ∞ while > whatever the typical org will use. The current default is probably a bit low (perhaps double it), but I don't see a lot to be gained by removing the limit entirely.